sdk/build.gradle

```diff
@@ -72,7 +72,7 @@ android {
 publish {
     def groupProjectID = 'com.karhoo'
     def artifactProjectID = 'nsdk-canary'
-    def publishVersionID = '1.0.12.01092020'
+    def publishVersionID = '1.0.13.09092020'
```

Comment:
I think we need to update it to `1.0.35.09092020`

Reply:
Done